README.md

```diff
@@ -82,7 +82,7 @@ Enjoy some useful utilities
 
 `NxtSupport::EnumHash` is a simple hash with indifferent access to organize a collection of enums. 
 Keys will be normalized to be underscore and downcase and access with [] is raising a KeyError in case there is 
-no value for the key. 
+no value for the key. It is close enough to flat `NxtRegistry`, but more lightweight. 
```

Comment:
Not sure about this tbh.

Reply:
I would also not phrase it this way. I mean it's not very close since EnumHash is meant for enums only and registry is more or less an implementation of the container pattern.